src/api/java/com/minecolonies/coremod/entity/IBaseEntityCitizen.java

```diff
@@ -0,0 +1,203 @@
+package com.minecolonies.coremod.entity;
+
+import com.minecolonies.coremod.client.render.BipedModelType;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.entity.Entity;
+import net.minecraft.entity.EntityAgeable;
+import net.minecraft.entity.INpc;
+import net.minecraft.entity.ai.EntityAITasks;
+import net.minecraft.entity.ai.EntitySenses;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.network.datasync.DataParameter;
+import net.minecraft.network.datasync.DataSerializers;
+import net.minecraft.network.datasync.EntityDataManager;
+import net.minecraft.util.EnumActionResult;
+import net.minecraft.util.EnumHand;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Vec3d;
+import net.minecraftforge.common.capabilities.ICapabilitySerializable;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Random;
+import java.util.UUID;
+
+public interface IBaseEntityCitizen extends ICommandSender, ICapabilitySerializable<NBTTagCompound>, INpc
```

Comment:
whatfor is the ICommandSender needed? so they can add chat messages?

Reply:
I guess so, this was on the original class so I just pulled it up into the interface.

That said I am not entirely happy with the api on the citizen level yet, but that is for another pr